addons/mail/static/src/widgets/form_renderer/form_renderer.js

```diff
@@ -31,6 +34,13 @@ FormRenderer.include({
         // Do not load chatter in form view dialogs
         this._isFromFormViewDialog = params.isFromFormViewDialog;
     },
+    /**
+     * @override
+     */
+    start() {
+        document.addEventListener('scroll', this._onScrollForm.bind(this), true);
```

Comment:
There is a syntax in widget to handle event based on class name, that doesn't work for this case?

Reply:
Actually we required scroll event on ".o_content" class. but it is actually a base template. and with event we can not have any custom scroll event based on class.